### PR TITLE
Initial api endpoint

### PR DESCRIPTION
**Pull request by zyunlam**

### CODE REVIEW

**Comment by SnehaSinghSS** [comment]
This is a good start for the recipe API endpoint, and as we go we can connect with Michael and figure out how to communicate with the database to get it fully functional

File: Front-End/login-signup/package-lock.json
Comment (mpkarpov-ui) [inline Front-End/login-signup/package-lock.json:1]: Gitignore this

File: compose.yml
Comment (mpkarpov-ui) [inline compose.yml:1]: Might need a rebase

**Review by mpkarpov-ui — CHANGES_REQUESTED** [review]
API changes look good to me but the delta is a lil silly rn

**Pull request merged**